common-mainnet.env

```diff
@@ -0,0 +1,4 @@
+ETHEREUM_NODE_URL=https://mainnet.infura.io/v3/9408f47dedf04716a03ef994182cf150
```

Comment:
Shouldn't we be putting these Infura project IDs as Travis variables?

Reply:
It's not really important as it's not really a secret. 
If we did this it would require manual exports on the local machine. If Infura ever starts rate-limiting/charging for accounts we might have to switch to a secret ID.

Reply:
> If Infura ever starts rate-limiting/charging for accounts we might have to switch to a secret ID.

I suppose they eventually will, after "to the moon"